src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -1138,3 +1163,69 @@ export const getStatus = (response) => {
     ? response.errors[0].status
     : 'unknown';
 };
+
+// Series of utility functions to sort claims and appeals by last updated date
+/**
+ * 
+ * @param {Object} appeal
+ * @returns {string}
+ */
+const getAppealDate = (appeal) => {
+  const { events } = appeal.attributes;
+  const dateString = (events && events.length)
+    ? events[events.length - 1].date
+    : '0';
+  return dateString;
+};
+
+/**
+ * 
+ * @param {Object} claim
+ * @returns {string}
+ */
+const getClaimDate = (claim) => {
+  const { phaseChangeDate } = claim.attributes;
+  const dateString = phaseChangeDate || '0';
+  return dateString;
+};
+
+/**
+ * 
+ * @param {Object} item
+ * @returns {string}
+ */
+const getDate = (item) => {
+  if (!item.attributes) {
+    return '0';
+  }
+
+  return (item.type === APPEAL_V2_TYPE)
+    ? getAppealDate(item)
+    : getClaimDate(item);
+};
+
+/**
+ * 
+ * @param {Object} item1
+ * @param {Object} item2
+ * @returns {-1|1|0}
+ */
+export function sortByLastUpdated(item1, item2) {
+  const lastUpdatedDate1 = getDate(item1);
+  const lastUpdatedDate2 = getDate(item2);
```

Comment:
Might it make more sense to use `moment` up here to avoid parsing each date twice?